src/frontend/screens/Library/components/GameCard/index.tsx

```diff
@@ -63,6 +63,24 @@ const GameCard = ({
   isRecent = false,
   gameInfo: gameInfoFromProps
 }: Card) => {
+  const [visible, setVisible] = useState(false)
+
+  useEffect(() => {
```

Comment:
start as invisible, make visible when the `visible-cards` event is fired and this card is in the list of visible cards